buildbot/deploy

```diff
@@ -38,13 +39,16 @@ elif [[ "${branch}" =~ ^refs/pull/([0-9]+)/ ]]; then
 
 	# First get the GitHub username of the Pull Request author.
 	pr_number=${BASH_REMATCH[1]}
-	username=`get_pr_username ${pr_number}`
+	pr_data=`get_pr_data ${pr_number}`
+	username=`get_pr_username "${pr_data}"`
+	labels=`get_pr_labels "${pr_data}"`
 
 	# Check if this user is either part of the developer team or the additional group of trusted people.
-	if ! { check_team_membership ${username} ${developer_group_id} || check_team_membership ${username} ${trusted_group_id}; }; then
+	if ! { check_team_membership ${username} ${developer_group_id} || check_team_membership ${username} ${trusted_group_id} || check_pr_label "${labels}" ${deploy_label}; }; then
```

Comment:
`check_pr_label` needs quotes around the labels, because they should each remain on their own line (in case of multiple labels)